src/pwr.ps1

```diff
@@ -1016,7 +1026,7 @@ switch ($Command) {
 						}
 						$Item.LastAccessTime = $Item.LastAccessTime
 						$Old = $Item.LastAccessTime -lt ((Get-Date) - (New-TimeSpan -Days $DaysOld))
-						if ($Old -and $PSCmdlet.ShouldProcess("${Name}:$Ver", 'remove pwr package')) {
```

Comment:
I have no clue what these were doing here but they produce verbose output.

Reply:
No. I need this here. It's what interacts with `-WhatIf` on an `rm` command

Reply:
Well, you don't need this code exactly how it was. I've updated to use `$WhatIfPreference` which should be equivalent to your wanted outcome.

Reply:
It is not equivalent to check the preference variable. That's just declaring default behavior. The shouldprocess method accounts for both and allows me to customize the output message.

Reply:
It is equivalent. If you pass `-WhatIf` then `WhatIfPreference` is true, otherwise it's false. The default value of `WhatIfPreference` is false, so you either use the default (false) or pass it explicitly as `-WhatIf:$false`.

Reply:
After reading the docs, it does in fact set the preference variable. 

BUT, I still don't want to change it because `ShouldProcess` also works with `-Confirm` and supports customizing the output. What's the issue here? Something about verbose output?

Reply:
Yeah, I've basically abandoned verbose output now and reverted the changes. Long live the information pipeline!